errata/src/main/scala/errata/types.scala

```diff
@@ -62,14 +62,16 @@ trait HandleTo[F[_], G[_], E] {
       fa: F[A]
   )(implicit F: Functor[F], G: Applicative[G]): G[Option[A]] =
     handle(F.map(fa)(_.some))(_ => None)
-
```

Comment:
What do you think about leaving `attempt` in here, but adding a type parameter `E1 >: E` and then returning `G[Either[E1, A]]` so its covariant?

Reply:
Doh! yeah of course! :) have applied your suggestion in a41717a71b2921326ed0d8b3924d49a285e8dc82